ChangeSet
  1.1536.1.1 04/03/30 11:44:35 boutcher@brule.rchland.ibm.com +1 -0
  fix pci_dma_mapping_error reference in rpa_vscsi.c

  drivers/scsi/ibmvscsi/rpa_vscsi.c
    1.14 04/03/30 11:43:53 boutcher@brule.rchland.ibm.com +1 -1
    fix pci_dma_mapping_error reference

diff -Nru a/drivers/scsi/ibmvscsi/rpa_vscsi.c b/drivers/scsi/ibmvscsi/rpa_vscsi.c
--- a/drivers/scsi/ibmvscsi/rpa_vscsi.c	Tue Mar 30 23:41:34 2004
+++ b/drivers/scsi/ibmvscsi/rpa_vscsi.c	Tue Mar 30 23:41:34 2004
@@ -169,7 +169,7 @@
 					  queue->size * sizeof(*queue->msgs),
 					  PCI_DMA_BIDIRECTIONAL);
 
-	if (pci_dma_error(queue->msg_token))
+	if (pci_dma_mapping_error(queue->msg_token))
 		goto map_failed;
 
 	rc = plpar_hcall_norets(H_REG_CRQ,
.........................................................................
# vim: syntax=diff

